mattermost/mattermost-plugin-github
# [MM-356] Add feature to subscribe to release and workflow events

#### Summary
- Added feature to publish workflow failure events

#### Screenshot 
![image](https://github.com/mattermost/mattermost-plugin-github/assets/100013900/43c9d62f-2f18-4b26-98bd-57ea1eb945cb)

#### Ticket Link
Fixes #722 
Fixes #565

#### What to test?
- Update the webhook for the repo to receive notifications for workflow events
- Create a subscription for a repo with a subscription enabled for workflow events
- Create workflow in which a particular jobs fail
- Check for the event received in the subscribed channel

#### Checklist
